webdev/nextjs/webdev.sh

```diff
@@ -131,6 +131,6 @@ case "$1" in
         cd azion/cells-site-template || exit $?
         # Publish only assets
         azion-framework-adapter publish --config ../kv.json \
-                               --only-assets --assets-dir ../../out
+                               --only-assets --assets-dir ../../out || exit $?
         echo "{}" > ./args.json;;
```

Comment:
Probably we should move this command `echo "{}" > ./args.json` to build step (after line 121).